scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -275,6 +308,11 @@ private void start(final InetSocketAddress bindAddress) throws IOException {
 		timer = Executors.newSingleThreadScheduledExecutor(
 				new DaemonThreadFactory("DTLS RetransmitTask-", NamedThreadFactory.SCANDIUM_THREAD_GROUP));
 
+		if (executor == null) {
+			// use a thread pool with max 6 * #CPU cores
+			executor = new StripedExecutorService(Runtime.getRuntime().availableProcessors() * 6);
```

Comment:
Hard to guess, because blocking time depends e.g. on the PSKStore implementation. I am not sure if we have blocking time when we use the default in memory PSK store.